packages/rpc/src/__tests__/rpc-request-coalescer-test.ts

```diff
@@ -110,7 +110,7 @@ describe('RPC request coalescer', () => {
                     return await new Promise((resolve, reject) => {
                         transportResponsePromise = resolve;
                         signal?.addEventListener('abort', (e: AbortSignalEventMap['abort']) => {
-                            reject((e.target as AbortSignal).reason);
+                            reject(new Error((e.target as AbortSignal).reason));
```

Comment:
I think the aim of the tests is to demonstrate that the `reason` gets thrown as-is. All this conversion does is to get in the way of that.